test/ship_test.rb

```diff
@@ -27,4 +27,27 @@ def test_it_has_a_length
   def test_it_can_have_a_different_length
     assert_equal 2, @destroyer.length
   end
+
+  def test_it_has_health
+    assert_equal 3, @cruiser.health
+  end
+
+  def test_it_knows_if_it_knows_its_sunk
+    assert_equal false, @cruiser.sunk?
+
+    @cruiser.hit
+    assert_equal false, @cruiser.sunk?
+
+    @cruiser.hit
+    assert_equal false, @cruiser.sunk?
+
+    @cruiser.hit
+    assert_equal true, @cruiser.sunk?
+  end
+
+  def test_it_can_get_hit
```

Comment:
I think the test_it_can_get_hit test should be moved above test_it_knows_if_it_knows_its_sunk because that test invokes/uses the .hit methods.